service/controller/catalog/resource/appcatalogentry/create.go

```diff
@@ -230,8 +230,7 @@ func (r *Resource) getDesiredAppCatalogEntryCR(ctx context.Context, cr *v1alpha1
 			AppVersion: e.AppVersion,
 			Catalog: v1alpha1.AppCatalogEntrySpecCatalog{
 				Name: cr.Name,
-				// Namespace will be empty until appcatalog CRs become namespace scoped.
-				Namespace: "",
+				Namespace: cr.Namespace,
```

Comment:
`AppCatalogEntry` CRs should be located in the same namespace as the `Catalog` CR.